install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+# FIXME: change to latest version after lightning fix issue
+# https://github.com/pingcap/tidb-lightning/issues/277
+TAG="v3.1.0-beta.1"
```

Comment:
is it better to always use latest lightning?

Reply:
an available lightning is enough, this test aimed to test dumpling not lightning